vizro-core/docs/pages/user-guides/data.md

```diff
@@ -268,3 +265,81 @@ data_manager["slow_expire_data"].timeout = 60 * 60
 data_manager["no_expire_data"] = load_iris_data
 data_manager["no_expire_data"].timeout = 0
 ```
+
+### Parametrize data loading
+
+You can supply arguments to your dynamic data loading function that can be modified from the dashboard. For example, if you are handling big data then you might like to load on demand only one chunk of the data.
```

Comment:
What would the arguments reflect if you continue your example. I think you need to say something more like, you can use an argument to specify the number of entries or size of chunk of data.

Reply:
How about this one?

```suggestion
You can supply arguments to your dynamic data loading function that can be modified from the dashboard. For example, if you are handling big data you can control the number of entries or the size of the data chunk in your dynamic data loading function through the dashboard, enabling you to load one large data chunk on demand.

```